cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -97,6 +97,7 @@ static const char *nvidia_globs[] = {
 	"libXvMCNVIDIA.so*",
 	"libXvMCNVIDIA_dynamic.so*",
 	"libcuda.so*",
+	"libcudart.so*",
```

Comment:
FYI opengl interface already allows accessing this library:
https://github.com/snapcore/snapd/blob/8378e64cf0931127cad078822c6fe94d983bb659/interfaces/builtin/opengl.go#L39